sdks/java/nexmark/build.gradle

```diff
@@ -42,6 +42,7 @@ configurations {
 }
 
 dependencies {
+  compile 'com.google.cloud:google-cloud-bigquery:1.28.0'
```

Comment:
We don't need this anymore, right ? (Also, This dependency will result in a conflict till we upgrade gRPC, protobuf, and Spanner. See https://issues.apache.org/jira/browse/BEAM-4229.)

Reply:
Indeed, I thought I had removed it, but I may have missed it in the rebase process. Thanks for pointing out